### PR TITLE
Sandra merges event posters into create events, also fixed merge conflicts

### PR DESCRIPTION
**Pull request by ServingCoder**

### CODE REVIEW

File: code/noRAM/app/src/main/java/com/example/noram/AddEventCompleteActivity.java [hunk 1]
```diff
@@ -46,6 +50,18 @@ protected void onCreate(Bundle savedInstanceState) {
         );
         event.updateDBEvent();
 
+        //upload photo to event poster cloud storage with matching event id
+        //set db with event poster. Only perform this action if an image was passed in the bundle
+        //implemented by
+        if (eventBundle.getParcelable("imageUri") != null) {
+            Uri imageUri = eventBundle.getParcelable("imageUri");
+            //String jpg = imageUri.getLastPathSegment();
```
Comment (ColeDewis) [inline code/noRAM/app/src/main/java/com/example/noram/AddEventCompleteActivity.java]: are these commented out lines still needed?

File: code/noRAM/app/src/main/java/com/example/noram/OrganizerCreateEventFragment.java [hunk 2]
```diff
@@ -40,10 +60,25 @@ public class OrganizerCreateEventFragment extends Fragment implements DatePicker
...
-    AppCompatButton editStartDateTime;
-    AppCompatButton editEndDateTime;
+    private AppCompatButton editStartDateTime;
+    private AppCompatButton editEndDateTime;
     View createdView;
 
+    private Uri imageUri;
+
+    AppCompatButton editStartTime;
+    private AppCompatButton editEndTime;
```
Comment (ColeDewis) [inline code/noRAM/app/src/main/java/com/example/noram/OrganizerCreateEventFragment.java]: are editStartDateTime and editStartTime different buttons? they sound like they're duplicates, but i might be wrong

**Review by ColeDewis — CHANGES_REQUESTED** [review]
couple small comments, but the main thing is you need to fix the build - it's failing due to missing constraints on an element, if you look at build details you can see exactly what the issue is. once that's dealt with this looks fine - we will just have to do the downloading/viewing later, which is ok.

File: code/noRAM/app/src/main/java/com/example/noram/OrganizerCreateEventFragment.java (hunk 2 above)
Comment (ServingCoder) [reply code/noRAM/app/src/main/java/com/example/noram/OrganizerCreateEventFragment.java]: I forgot to remove these after the merge. we use private AppCompatButton editStartDateTime;
    private AppCompatButton editEndDateTime;

File: code/noRAM/app/src/main/java/com/example/noram/AddEventCompleteActivity.java (hunk 1 above)
Comment (ServingCoder) [reply code/noRAM/app/src/main/java/com/example/noram/AddEventCompleteActivity.java]: I removed commented out lines

**Review by ColeDewis — APPROVED** [review]
Thanks for the changes, LGTM now

**Pull request merged**